XRTK.Lumin/Packages/com.xrtk.lumin/Controllers/LuminHandDataConverter.cs

```diff
@@ -58,6 +57,24 @@ public HandData GetHandData()
             return updatedHandData;
         }
 
+        /// <summary>
+        /// Gets the magic leap hand reference for the given handedness.
+        /// </summary>
+        /// <param name="handedness">Handedness to convert.</param>
+        /// <returns>Magic Leap hand reference.</returns>
+        private static MLHand ToMagicLeapHand(Handedness handedness)
```

Comment:
Why remove the extension?

Reply:
Because it was only used in one place and I didn't think it was that useful tbh

Reply:
That and In don't think you can call `this` on an enum